index.html

```diff
@@ -0,0 +1,1209 @@
+
+                  <p>Servers who want a resource to inherit Authorizations (<cite><a href="#effective-acl-resource" rel="rdfs:seeAlso">Effective ACL Resource</a></cite>) from a container resource MUST NOT initialise the ACL resource that is associated with the resource with a representation. When an authorized HTTP <code>GET</code> or <code>HEAD</code> request targets an ACL resource without an existing representation, the server MUST respond with the <code>404</code> status code as per [<cite><a class="bibref" href="#bib-rfc7231">RFC7231</a></cite>].</p>
+
+                  <p>The <a href="#root-container">root container</a> MUST have an ACL resource with a representation. The ACL resource of the root container MUST include an Authorization allowing the <code>acl:Control</code> access privilege (<cite><a href="#acl-mode-control" rel="rdfs:seeAlso"><code>acl:Control</code></a></cite> access mode).</p>
+
+                  <p class="advisement">Source: <a href="https://github.com/solid/web-access-control-spec/issues/45" rel="cito:citesAsSourceDocument">issues/45</a></p>
+                </div>
+              </section>
+            </div>
+          </section>
+
+          <section id="authorization-rule" inlist="" rel="schema:hasPart" resource="#authorization-rule">
+            <h2 property="schema:name">Authorization Rule</h2>
+            <div datatype="rdf:HTML" property="schema:description">
+              <p><a href="#authorization">Authorization</a> (instance of <code>acl:Authorization</code>) is the most fundamental unit of access control describing access permissions granted to agents the ability to perform operations on resources. Authorizations are described with RDF statements and may express any information. This section describes the following characteristics of an Authorization: <a href="#access-objects" rel="cito:discusses">access objects</a> to specify what can be accessed, <a href="#access-modes" rel="cito:discusses">access modes</a> to specify permissions, and <a href="#access-subjects" rel="cito:discusses">access subjects</a> to specify who can access the objects.</p>
+
+              <section id="access-objects" inlist="" rel="schema:hasPart" resource="#access-objects">
+                <h3 property="schema:name">Access Objects</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>The <code>acl:accessTo</code> property value is used to check if access is allowed for a specific resource. When an Authorization includes the <code>acl:default</code> property value (the container resource in context), then access permissions be applied to the original requested resource.</p>
+
+                  <p>Inheriting Authorizations from the most significant container’s ACL resource is useful to avoid individually managing an ACL resource for each resource, as well as to define access control for resources that do not exist yet.</p>
```

Comment:
This seems to imply it's not possible to create an acl before creating the corresponding resource, is that intended?
A consequence of that could be that if you first create a resource and then quickly add an ACL to it, there is a (short) time in which the resource already exists but doesn't have an ACL yet. This is a potential security risk if the ACL of the resource gives less access to some parties than the reigning default one. So if you want to create a private resource in a public container, you would temporarily have to make the container private, then create the resource, then create the ACL of the resource, then set the container back to public.

Reply:
Intended. There are existing issues/discussions on this. Let's continue there. But for starters, WAC doesn't specify how to create an ACL resource for a resource that doesn't exist.

Reply:
Good point. I think we have discussed that earlier, it is something we should address.